test/dlc_tests/container_tests/bin/testTensorflow2Standalone

```diff
@@ -26,17 +27,22 @@ def check_tensorflow():
         sess = tf.compat.v1.Session(config=tf.compat.v1.ConfigProto(log_device_placement=False))
 
         with sess:
+            tf_version = tf.__version__
+            version_check = r"2.[0-3]+"
+            device_tag = ''
+            if(re.search(version_check, tf_version)):
+                device_tag = 'XLA_'
             if is_gpu:
                 # Use GPU device 0
-                with tf.device('/job:localhost/replica:0/task:0/device:XLA_GPU:0'):
+                with tf.device('/job:localhost/replica:0/task:0/device:{}GPU:0'.format(device_tag)):
                     x_mat = tf.constant([1.0, 2.0, 3.0, 4.0, 5.0, 6.0], shape=[2, 3], name='x_mat')
                     y_mat = tf.constant([1.0, 2.0, 3.0, 4.0, 5.0, 6.0], shape=[3, 2], name='y_mat')
                     z_mat = tf.matmul(x_mat, y_mat)
 
                 assert sess.run(z_mat).any(), "Error: GPU sanity test for TensorFlow failed."
             else:
                 # Use CPU device 0
-                with tf.device('/job:localhost/replica:0/task:0/device:XLA_CPU:0'):
+                with tf.device('/job:localhost/replica:0/task:0/device:{}CPU:0'.format(device_tag)):
                     x_mat = tf.constant([1.0, 2.0, 3.0, 4.0, 5.0, 6.0], shape=[2, 3], name='x_mat')
                     y_mat = tf.constant([1.0, 2.0, 3.0, 4.0, 5.0, 6.0], shape=[3, 2], name='y_mat')
                     z_mat = tf.matmul(x_mat, y_mat)
```

Comment:
It looks like both the if and else code paths are almost exactly identical except for the `tf.device` definition. Could you make "GPU", "CPU", and the "XLA_" prefix all parametrized on the value of `is_gpu` and `tf_version`, and only write this block of code once?

Something like this:
```
device_tag = ("XLA_" if re.match(pattern, tf_version) else "") + ("GPU" if is_gpu else "CPU")
with tf.device(f"/job:localhost/replica:0/task:0/device:{device_tag}:0"):
    x_mat = tf.constant([1.0, 2.0, 3.0, 4.0, 5.0, 6.0], shape=[2, 3], name='x_mat')
    y_mat = tf.constant([1.0, 2.0, 3.0, 4.0, 5.0, 6.0], shape=[3, 2], name='y_mat')
    z_mat = tf.matmul(x_mat, y_mat)
    assert sess.run(z_mat).any(), f"Error: {device_tag} sanity test for TensorFlow failed."
```

Reply:
I have ported the changes to the new pr 779, can you please close this pr, thanks.